src/server/game/Entities/Player/Player.cpp

```diff
@@ -1494,7 +1497,7 @@ bool Player::TeleportTo(uint32 mapid, float x, float y, float z, float orientati
                     InterruptNonMeleeSpells(true);
 
             //remove auras before removing from map...
-            RemoveAurasWithInterruptFlags(AURA_INTERRUPT_FLAG_CHANGE_MAP | AURA_INTERRUPT_FLAG_MOVE | AURA_INTERRUPT_FLAG_TURNING);
+            RemoveAurasWithInterruptFlags(AURA_INTERRUPT_FLAG_CHANGE_MAP | AURA_INTERRUPT_FLAG_TELEPORTED | AURA_INTERRUPT_FLAG_MOVE | AURA_INTERRUPT_FLAG_TURNING);
```

Comment:
As I said earlier - that flag is not related to teleport.

Reply:
I ensure you it is a different flag than CHANGE_MAP, if you watch the two videos you'll spot it (:
More, its name is explicit imo, and it is recognized by the core and correctly doing its job when I use it.

Reply:
@UltraNix after checking within Stone Harry's Spell Editor, these spells do not have aura interrupt flag TELEPORTED (se this [video](https://youtu.be/lOW207cbQME) : https://youtu.be/lOW207cbQME)

Reply:
@UltraNix I reverted this part of the PR. You're right, the case here might not be regular teleport. It seems more related to map id and/or map instance. Did not totally understand, but stopped messing with it.

Then the PR now is only one line (:

After testing it with my two spells it works correctly. 

STILL: as I only remove auras with interrupt flag TELEPORTED and no longer the others, should I keep int under the 100 yards condition ? Or is it better to be absolute about teleport and remove auras with interrupt flag TELEPORTED just above the 100 yards condition ? (second option is more logic to me, as the flag name only mentions teleporting, not "teleporting far" or whatever)

Here is one more video with the previous commit, removing auras with flag TELEPORTED within the 100 yards condition:
https://youtu.be/5Qe81YA63Ec

I'll try outside the 100 yards condition and post another video of it.

Reply:
@UltraNix outside the 100 yards condition, so removes auras with this interrupt flag whenever teleporting:
https://youtu.be/_dCTi4kMsqs